dist/fabric.js

```diff
@@ -28186,6 +28192,34 @@ fabric.Image.filters.BaseFilter.fromObject = function(object, callback) {
       return this;
     },
 
+    setPosition: function(position){
+      this.left = position.left;
+      this.top = position.top;
+      this.initDimensions();
+      this.setCoords();
+    },
+
+    setTemporaryPosition: function(position) {
+      this.originalPosition = {
+        left: this.left,
+        top: this.top,
+      };
+      this.usingTemporaryPosition = true;
+      this.setPosition(position);
+    },
+
+    restoreOriginalPosition: function() {
+      this.usingTemporaryPosition = false;
+      if (this.originalPosition) {
+        this.setPosition(this.originalPosition);
+      }
+    },
+
+    clearTemporaryPosition: function() {
+      this.usingTemporaryPosition = false;
+      this.originalPosition = undefined;
+    },
+
```

Comment:
Don't think you need to commit this. It will be compiled during Jenkins build.

Reply:
do we add this file to gitignore?

Reply:
I found it useful to link to these when working in classroom, and found it helpful to already have these compiled and updated. Jenkins will build this and include it in the NPM module but it won't update the repo. Not sure how you work on fabric within classroom these days... if there is no other workflow I'd recommend keeping these.

Reply:
You are right, Yuriy. Jenkins will give this to us. I just like having it as a sanity check when looking over the PR since all the changes are in one place. 

As the owner of amplify/fabricjs - I say it stays!